src/editor/pages/api/token/create.ts

```diff
@@ -0,0 +1,13 @@
+import * as FlooqApi from '../../../helper/flooq-api'
+
+const handler = async ( req: any, res: any ): Promise<void> => {
+  req.body = JSON.parse( req.body )
+  const token = {
+    name: req.body.name,
+    value: req.body.value
+  }
```

Comment:
Why do you parse the body, create a new object and then stringify it again? Seems a bit excessive :D

Reply:
To extract certain attributes from it. How else am I supposed to do this?